### DISCUSSION TITLE
posts/zabbix-alertmanager/

### DISCUSSION THREAD

**Discussion by giscus** [liwanggui/liwanggui.github.io#107] [Announcements]
# posts/zabbix-alertmanager/

由于 zabbix 自身并不支持告警分组及收敛功能，经常会出现告警信息轰炸的情况，为了解决这个情况可以使用 Alertmanager 来管理 zabbix 的告警消息 Alertmanager 特性Alertmanage

https://liwanggui.com/posts/zabbix-alertmanager/

**Comment by xiarunjie** [comment]
尊敬的作者，你会考虑使用zabbix的webhook来发送告警么？感觉这样的依赖更少。

**Reply by liwanggui** [reply]
已经很久没用 zabbix 了，如使用 webhook 方式，得写程序去解析推送过来的数据及发送消息
